tests/manager/api/admin/controller/test_work_editor.py

```diff
@@ -74,64 +75,6 @@ def work_fixture(
 
 
 class TestWorkController:
-    def test_details(self, work_fixture: WorkFixture):
-        [lp] = work_fixture.english_1.license_pools
-
-        lp.suppressed = False
-        with work_fixture.request_context_with_library_and_admin("/"):
-            response = work_fixture.manager.admin_work_controller.details(
-                lp.identifier.type, lp.identifier.identifier
-            )
-            assert 200 == response.status_code
-            feed = feedparser.parse(response.get_data())
-            [entry] = feed["entries"]
-            suppress_links = [
-                x["href"]
-                for x in entry["links"]
-                if x["rel"] == "http://librarysimplified.org/terms/rel/hide"
-            ]
-            unsuppress_links = [
-                x["href"]
-                for x in entry["links"]
-                if x["rel"] == "http://librarysimplified.org/terms/rel/restore"
-            ]
-            assert 0 == len(unsuppress_links)
-            assert 1 == len(suppress_links)
-            assert lp.identifier.identifier in suppress_links[0]
-
-        lp.suppressed = True
-        with work_fixture.request_context_with_library_and_admin("/"):
-            response = work_fixture.manager.admin_work_controller.details(
-                lp.identifier.type, lp.identifier.identifier
-            )
-            assert 200 == response.status_code
-            feed = feedparser.parse(response.get_data())
-            [entry] = feed["entries"]
-            suppress_links = [
-                x["href"]
-                for x in entry["links"]
-                if x["rel"] == "http://librarysimplified.org/terms/rel/hide"
-            ]
-            unsuppress_links = [
-                x["href"]
-                for x in entry["links"]
-                if x["rel"] == "http://librarysimplified.org/terms/rel/restore"
-            ]
-            assert 0 == len(suppress_links)
-            assert 1 == len(unsuppress_links)
-            assert lp.identifier.identifier in unsuppress_links[0]
-
-        work_fixture.admin.remove_role(
-            AdminRole.LIBRARIAN, work_fixture.ctrl.db.default_library()
-        )
-        with work_fixture.request_context_with_library_and_admin("/"):
-            pytest.raises(
-                AdminNotAuthorized,
-                work_fixture.manager.admin_work_controller.details,
-                lp.identifier.type,
-                lp.identifier.identifier,
-            )
-
```

Comment:
The `details` endpoint is not going away, so these tests will be needed, just in a different form. 

It is okay for them to be removed for the moment, though. I will add them back in some form as part of work on PP-1318.